apps/ledger-live-mobile/src/logic/postOnboarding/PostOnboardingProviderWrapped.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { PostOnboardingProvider } from "@ledgerhq/live-common/postOnboarding/PostOnboardingProvider";
+import { getPostOnboardingAction, getPostOnboardingActionsForDevice } from ".";
+import { useNavigateToPostOnboardingHubCallback } from "./useNavigateToPostOnboardingHubCallback";
+
+type Props = {
+  children?: React.ReactNode | undefined;
+};
+
+const PostOnboardingProviderWrapped: React.FC<Props> = ({ children }) => {
+  const navigateToPostOnboardingHub = useNavigateToPostOnboardingHubCallback();
+  return (
+    <PostOnboardingProvider
+      navigateToPostOnboardingHub={navigateToPostOnboardingHub}
+      getPostOnboardingAction={getPostOnboardingAction}
+      getPostOnboardingActionsForDevice={getPostOnboardingActionsForDevice}
+    >
```

Comment:
are those props always the same? or is `PostOnboardingProvider` ever used with different props?
can't we just put them inside `PostOnboardingProvider` directly and have one less layer of components?

Reply:
For now it's always the same but later on LLD will specify its own post onboarding features (which will differ at least because for instance the navigation params will be different)